cvat/apps/engine/serializers.py

```diff
@@ -396,6 +397,7 @@ def validate(self, attrs):
         if 'start_frame' in attrs and 'stop_frame' in attrs \
             and attrs['start_frame'] > attrs['stop_frame']:
             raise serializers.ValidationError('Stop frame must be more or equal start frame')
+
```

Comment:
Probably, we still need to validate the pattern expression.

Reply:
@zhiltsov-max, Could you please provide pattern example when we need to handle it? (If we use regex I agree that validation is needed). I guess It is not necessary anymore.

Reply:
My guess is that we could check for invalid characters in the pattern and produce a meaningful error message in case of problems. It's up to you to decide.